tests/ReflectionClosure1Test.php

```diff
@@ -276,7 +276,7 @@
 
 test('interpolation1', function () {
     $f1 = function () {
-        return "${foo}${bar}{$foobar}";
+        return "{$foo}{$bar}{$foobar}";
```

Comment:
This change is definitely not wanted. We need to test the old synax works properly, even though it's deprecated.

Reply:
Why you want that? The new syntax works in PHP >= 7.4: https://3v4l.org/4MWFX